domain/BUILD.bazel

```diff
@@ -123,6 +124,7 @@ TEST_DEPS = [
     ":testing",
     "//data/src/main/java/org/oppia/android/data/backends/gae/model",
     "//data/src/main/java/org/oppia/android/data/persistence:cache_store",
+    "//domain/src/main/java/org/oppia/android/domain/feedbackreporting:report_schema_version",
```

Comment:
The module also needs to be included as a dep @deepanshu731. I think that's why you're getting an error.

You can verify this is correct locally by trying to run the test that's failing in CI:

```sh
bazel test //domain:src/test/java/org/oppia/android/domain/feedbackreporting/ReportSchemaVersionTest
```

If that builds & passes locally, you've got the deps correct.

Reply:
thanks @BenHenning , test passed locally